src/Schema/SchemaContainer.php

```diff
@@ -171,10 +169,33 @@ protected function hasPath(Connection $connection, $parent)
     }
 
     /**
+     * Add connection to collection.
+     *
+     * @param  string $name
+     * @param  string $namespace
+     * @return Field
+     */
+    public function connection($name, $namespace)
+    {
+        $edgeType = $this->createField($name.'Edge', $namespace);
+
+        $this->types->push($edgeType);
+
+        $connectionType = $this->createField($name.'Connection', $namespace);
```

Comment:
To go along with the convention, this should probably be:

```php
$connectionType = $this->createField($name, $namespace);
```

So when someone registers the connection in the schema file:

```php
Relay::connection('posts', 'SomeClassName');
```

They can use the name they registered in their type:

```php
'type' => GraphQL::type('posts')
```

instead of 

```php
'type' => GraphQL::type('postsConnection');
```